examples/udiv0Test.s

```diff
@@ -0,0 +1,4 @@
+; TinyRAM V=1.000 W=16 K=16
+mov r2, 5
+udiv r1, r2, 2
+answer r1
```

Comment:
This file hasn't a corresponding test.

Reply:
Wasn't the intent of the test dividing by 0?

Reply:
This has been fixed and the test has been added to the Spec file.